File: 백준/Silver/1929. 소수 구하기/소수 구하기.java
import java.io.BufferedReader;
import java.io.IOException;
import java.io.InputStreamReader;

public class Main {
	
	public static boolean isPrime(int n) {
		for(int p = 2; p < (int)Math.sqrt(n) + 1; p++) {
			if (n % p == 0) {
				return false;
			}
		}
		return true;
	}
	
	public static void main(String[] args) throws IOException{
		BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
		String[] nums = br.readLine().split(" ");
		int[] num = new int[2]; 
		for(int i = 0; i < 2; i++) {
			num[i] = Integer.parseInt(nums[i]);
		}
		
		int start = num[0];
		if (start == 1) start = 2;
		
		for(int i = start; i < num[1] + 1; i++) {
			if (isPrime(i)) {
				System.out.println(i);
			}
		}
	}
}

File: 백준/Silver/1929. 소수 구하기/소수 구하기.py
import math

start, end = map(int, input().split())

def is_prime(num):
    for i in range(2, int(math.sqrt(num)) + 1):
        if num % i == 0:
            return False
    return True

if start == 1:
    start = 2
    
for i in range(start, end+1):
    if is_prime(i):
        print(i)
